.github/workflows/Build.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       run: make BUILD=debug
 
     - name: Examples
-      run: make examples BUILD=debug
+      run: make examples
```

Comment:
Why remove `BUILD=debug`? I think this is more useful here as it will catch my sloppy code ;)

Reply:
Oh, wait I see this is only for the examples, please ignore